tools/open_dataset_tool/objects365.py

```diff
@@ -0,0 +1,194 @@
+# Copyright (C) 2024 Nota Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ----------------------------------------------------------------------------
+
+import os
+import time
+import json
+import torch
+import shutil
+import argparse
+import subprocess
+from pathlib import Path
+from tqdm import tqdm
+from multiprocessing import Pool
+
+DEFAULT_DATA_DIR = './data'
+DOWNLOAD_DIR = './data/download'
```

Comment:
I think Objects365 dataset has so many data split to download. How about changing download target path to `./data/download/objects365`?

```suggestion
DOWNLOAD_DIR = './data/download/objects365'
```